src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -8,7 +8,9 @@ import {atom, useRecoilState} from 'recoil';
 import AnonymousMessageRepo from '../../../service/repo/anonymousMessageRepo';
 import ChannelList from '../../../database/schema/ChannelListSchema';
 import SignedMessageRepo from '../../../service/repo/signedMessageRepo';
-import UseChatUtilsHook from '../../../../types/hooks/screens/useChatUtilsHook.types';
+import UseChatUtilsHook, {
+  ContactScreenPayload
+} from '../../../../types/hooks/screens/useChatUtilsHook.types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `UseChatUtilsHook` and `ContactScreenPayload` has been modified. It seems like you are destructuring the imported types from `useChatUtilsHook.types`. This is a good practice as it makes it clear what exactly is being imported from the module, improving readability.
```diff
- import UseChatUtilsHook from '../../../../types/hooks/screens/useChatUtilsHook.types';
+ import UseChatUtilsHook, {
+   ContactScreenPayload
+ } from '../../../../types/hooks/screens/useChatUtilsHook.types';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->